dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -53,6 +56,9 @@
     private static final TraceComponent tc = Tr.register(CacheStoreService.class);
     
     private Map<String, Object> configurationProperties;
+    private static final String BASE_PREFIX = "properties";
+    private static final int BASE_PREFIX_LENGTH = BASE_PREFIX.length();
+    private static final int TOTAL_PREFIX_LENGTH = BASE_PREFIX_LENGTH + ".0.".length();
```

Comment:
Replace with +3 and a comment indicating that 3 is the length of ".0." rather than constructing a string for the sole purpose of counting its characters.

Reply:
Good catch, I copied this from the JDBC code without much thought, not sure why we're doing that there.  I'll update to use 3.